### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by he3als**
### Type
- [ ] Rewrite already written documentation.
- [x] Add/remove new pages.

### Questions
- [x] Did you preview on your local machine beforehand?
- [x] Did you follow our [commit handbook](https://github.com/Atlas-OS/docs/blob/master/.github/CONTRIBUTING.md#commit-message)?

### Describe your pull request
Adds contribution guidelines (see [here](https://contribution-guidelines.atlasos-docs-enq.pages.dev/contributions/)) for general contributions to the Atlas repos.

### CODE REVIEW

**Comment by he3als** [comment]
Before this is merged, there should be reviews and the Atlas repos should be setup accordingly with automatic pull requests and the `dev` branches.

**Comment by whallin** [comment]
> Before this is merged, there should be reviews and the Atlas repos should be setup accordingly with automatic pull requests and the `dev` branches.

Are all the documentation changes done otherwise? I was considering if the PR should be drafted or not in case the documentation side of things aren't completed yet.

**Comment by he3als** [comment]
The documentation I think is personally completed, but I am open to any changes of course.

**Comment by JayXTQ** [comment]
We should include the GitHub workflows in this PR before we push if we want to include them

**Comment by JayXTQ** [comment]
Anyone willing to ChatGPT it?

**Comment by he3als** [comment]
> We should include the GitHub workflows in this PR before we push if we want to include them

For this repo, yes. The other repos will require their own.

> Anyone willing to ChatGPT it?

It seems generally okay I think, making ChatGPT rewrite it is probably not necessary.

**Comment by whallin** [comment]
> We should include the GitHub workflows in this PR before we push if we want to include them

What workflows will be in this repository?

**Comment by he3als** [comment]
The automation for making a PR merging `dev` and `main` each week.

**Comment by whallin** [comment]
> The automation for making a PR merging `dev` and `main` each week.

Ah, yes, okay. Maybe for the sake of it, if anyone feels it, add it to this PR. Otherwise, save it for another request.

**Comment by JayXTQ** [comment]
> It seems generally okay I think, making ChatGPT rewrite it is probably not necessary.

@he3als I meant make ChatGPT write the workflow file

**Comment by he3als** [comment]
Oh sorry, well, you will probably have to mostly write it yourself, ChatGPT could give a base though I guess. I will try to make it.

**Review by Xyueta — APPROVED** [review]

**Comment by he3als** [comment]
> > The automation for making a PR merging `dev` and `main` each week.
> 
> Ah, yes, okay. Maybe for the sake of it, if anyone feels it, add it to this PR. Otherwise, save it for another request.

@whallin Completed, GitHub Actions should be enabled for this repository and this should be ticked:

image

https://github.com/Atlas-OS/docs/commit/7b853c465641671ce596b12e000933869a0cb8b9

**Comment by he3als** [comment]
@JayXTQ

File: mkdocs.insiders.yml
Comment (whallin) [inline mkdocs.insiders.yml:127]: Forums*, capitalized F. I'm picky.

File: mkdocs.yml
Comment (whallin) [inline mkdocs.yml:125]: Forums*, capitalized F. I'm picky. (same as insiders config)

**Review by whallin — COMMENTED** [review]
Might be too picky, otherwise it all looks 👌🏻

**Review by whallin — APPROVED** [review]
👌🏻

**Review by SyrmGIT — APPROVED** [review]
Spot on.

**Comment by he3als** [comment]
Good to merge?

**Comment by whallin** [comment]
> Good to merge?

Wait for a last pair of eyes. Quite a big change, don't want to miss any grammatical or factual errors.

**Comment by he3als** [comment]
I'll also edit the contribution guide in this repo.

**Review by Xyueta — DISMISSED** [review]

**Review by PencilNavigator — APPROVED** [review]
lgtm

**Review by JayXTQ — APPROVED** [review]
Doesn’t look too horrible

**Review by whallin — APPROVED** [review]
👍🏻

**Comment by JayXTQ** [comment]
Waiting for @he3als to give the go-ahead

**Comment by he3als** [comment]
I'll edit the contribution guidelines for this repo in another pull request.

**Pull request merged**
